Softeq.XToolkit.Remote/Primitives/RequestOptions.cs

```diff
@@ -19,38 +22,50 @@ public class RequestOptions
         public const int DefaultTimeout = 20;
 
         /// <summary>
-        ///     Gets or sets default RequestOptions object. If defined, it will be used when no options are specified.
+        ///     Gets or sets default <see cref="RequestOptions"/> object.
+        ///     If defined, it will be used when no options are specified.
         /// </summary>
-        public static RequestOptions? DefaultRequestOptions { get; set; }
+        public static RequestOptions? DefaultRequestOptions { get; set; } = default!;
 
         /// <summary>
-        ///     Gets default should retry condition. Default value is null.
+        ///     Gets default should retry condition.
+        ///     Default value is "() => true" - always retry.
         /// </summary>
         public static Func<Exception, bool> DefaultShouldRetry { get; } = _ => true;
 
         /// <summary>
-        ///     Gets default cancellation token. Default value is None.
+        ///     Gets default cancellation token.
+        ///     Default value is <see cref="P:System.Threading.CancellationToken.None"/>.
```

Comment:
As I can see, this value is always CancellationToken.None, so maybe it makes sense to change the documentation a little bit: 
```suggestion
        ///     The value is <see cref="P:System.Threading.CancellationToken.None"/>.
```